src/DotLiquid/Context.cs

```diff
@@ -524,14 +524,15 @@ private bool TryGetVariable(IEnumerator<string> partsEnumerator, out object vari
 
                 // If object is a KeyValuePair, we treat it a bit differently - we might be rendering
                 // an included template.
-                if (IsKeyValuePair(@object) && (part.SafeTypeInsensitiveEqual(0L) || part.Equals("Key")))
+                var isKvp = IsKeyValuePair(@object);

```

Comment:
```suggestion
                var isKeyValuePair = IsKeyValuePair(@object);
```
Avoid use of uncommon acronyms.